doc/source/index.rst

```diff
@@ -280,7 +280,7 @@ Getting Involved
 
 .. toctree::
    :maxdepth: -1
-   :caption: RayServe
+   :caption: Ray Serve
 
    rayserve/overview.rst
```

Comment:
These need to be updated to `serve/...`

Reply:
I think I got them when you said this, they're correct in the current PR right?